the-blue-alliance-ios/Core Data/TBA.xcdatamodeld/TBA.xcdatamodel/contents

```diff
@@ -113,7 +113,7 @@
     <entity name="EventRanking" representedClassName="EventRanking" syncable="YES" codeGenerationType="class">
         <attribute name="dq" optional="YES" attributeType="Integer 16" usesScalarValueType="NO" syncable="YES"/>
         <attribute name="extraStatsNames" optional="YES" attributeType="Transformable" customClassName="[String]" syncable="YES"/>
-        <attribute name="extraStatsValues" optional="YES" attributeType="Transformable" customClassName="[Int]" syncable="YES"/>
+        <attribute name="extraStatsValues" optional="YES" attributeType="Transformable" customClassName="[NSNumber]" syncable="YES"/>
```

Comment:
Are we going to have to write custom migrations for these, or does the `customClassName` stuff work automagically and since the underlying data hasn't changed (I think)

Reply:
Looking at tests for this as well - we're using all Int's. We should try to add some test coverage to make sure we support different types.

Reply:
I can look into tests, but I have no idea how to identify / test for issues with core data migrations. If you could point me in the right direction that'd be great 👍

Reply:
actually unsure how to add tests around this as well, mostly because I think this logic should be moved to EventRanking.swift instead of RankingCellViewModel.swift

Reply:
Re: Core Data migration - the easiest way to test is to launch the app from master, load some data (go to a rankings view), then checkout your branch and build/run again. If the app crashes on launch, we're gonna have a problem.

I can test this too - I tried to last night, but your branch didn't build for me. Then I got distracted...